src/main/java/seedu/address/logic/commands/DeleteTransactionCommand.java

```diff
@@ -50,6 +51,7 @@ public CommandResult execute(Model model) throws CommandException {
         Transaction transactionToRemove = transactions.get(index.getZeroBased());
         transactions.remove(index.getZeroBased());
         selected.updateBalance(transactionToRemove.getAmount() * -1);
+        selected.updateTransactions(transactions);
```

Comment:
This line is causing some problems.
For example: Person 1 has 5 transactions
1. `findt 1 keywords` lists 3 out of 5 transactions for Person 1
2. `deletet 2` deletes the 2nd transaction out of 3
3. `list` + `listt 1` only lists 2 transactions

because when this line updates Person 1's transaction list, it only takes the result of `findt` - 1 deleted transaction.